frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -275,7 +275,7 @@ export default ({
         if (alreadyAdded >= 0) {
           emoticons[emoticon].splice(alreadyAdded, 1)
           if (emoticons[emoticon].length === 0) {
-            delete this.messages[emoticon]
+            delete this.messages[index].emoticons[emoticon]
```

Comment:
@pieer, I think you need to make a new branch from `chat-contract-messaging` branch and make pull request to `chat-contract-messaging` branch. Actually, making reactions is not implemented in the `main` branch.
I think this is my mistake that I didn't tell you before. You couldn't know.